docs/source/guide/pec-5-theory.myst

```diff
@@ -0,0 +1,156 @@
+to the error mitigation cost associated to the gate $\mathcal G_i$.
+
+***Note:** In principle, the gate index "$i$" in the noisy operations $\mathcal O_{i, \alpha}$ could be dropped.
+However, we keep it to explicitly define gate-dependent basis of implementable operations, consistently with 
+the  structure of the `OperationRepresentation` class discussed in [What additional options are
+available in PEC?](pec-3-options.myst).*
+
+
+## Error cancellation
+
+The aim of PEC is estimating the ideal expectation value of some observable $A=A^\dagger$ with respect to 
+the quantum state prepared by an ideal circuit of interest $\mathcal U$ acting on some initial  reference 
+state $\rho_0$ (typically $\rho_0= |0\dots 0 \rangle \langle 0 \dots 0 |$).
+
+Replacing each gate $\mathcal G_i$ with its noisy representation, we can express the ideal expectation
+value as a linear combination of noisy expectation values:
+
+$$
+\langle A \rangle_{\rm ideal}= {\rm tr}[A \mathcal U (\rho_0)] = 
+\sum_{\vec{\alpha}} \eta_{\vec{\alpha}} \langle A_{\vec{\alpha}}\rangle_{\rm noisy}
+$$ 
+
+where 
+
+$$ 
+\eta_{\vec{\alpha}} := \eta_{t, \alpha_t} \dots \eta_{2, \alpha_2} \eta_{1,\alpha_1}, 
+\quad  \langle A_{\vec{\alpha}}\rangle_{\rm noisy} :=  {\rm tr}[A \Phi_{\vec{\alpha}}(\rho_0)],
+\quad \Phi_{\vec{\alpha}} := \mathcal O_{t, \alpha_t} \circ \dots \circ \mathcal O_{2, \alpha_2} \circ \mathcal O_{1, \alpha_1}.
+$$
+
+
+The coefficients $\{ \eta_{\vec{\alpha}} \}$ form a quasi-probability distribution
+for the global circuit over the noisy circuits. Indeed it is easy to check that, at the level of super-operators,
+we have:
+
+$$ \mathcal U =  \sum_{\vec{\alpha}} \eta_{\vec{\alpha}} \Phi_{\vec{\alpha}}. $$
+
+The one-norm $\gamma$ of the global quasi-probability distribution is the product of those of the gates:
+
+$$
+\sum_{\vec \alpha} \eta_{\vec{\alpha}}=1,  \qquad  \gamma = \sum_{\vec{\alpha}} |\eta_{\vec \alpha}| = \Pi_{i=1}^{t} \gamma_i.
+$$
+
+All the noisy expectation values $\langle A_{\vec{\alpha}}\rangle_{\rm noisy}$ can be directly measured with 
+a noisy backend since they only require circuits composed of implementable noisy operations.
+In principle, by combining all the noisy expectation values, one could compute the ideal result $\langle A \rangle_{\rm ideal}$.
+Unfortunately this approach requires the execution of an exponentially large number of circuits and 
+which is typically unfeasible. 
+
+An important fact at the basis of PEC is that, for weak noise, only a small number of noisy expectation values actually
+contribute to the linear combination because many of the coefficients $\eta_{\vec \alpha}$ are negligible.
+For this reason, it is more efficient to estimate $\langle A \rangle_{\rm ideal}$ using an
+ ([importance-sampling](https://en.wikipedia.org/wiki/Importance_sampling)) Monte Carlo approach as described in the next section.
```

Comment:
```suggestion
 [importance-sampling](https://en.wikipedia.org/wiki/Importance_sampling) Monte Carlo approach as described in the next section.
```

Reply:
just a suggestion. The "an" read strange without the parenthesis. Or maybe could be a(n)